### PR TITLE
Displays badge for reciprocal member accounts (AIC-614)

### PR DESCRIPTION
**Pull request by sam33rdhakal**

### CODE REVIEW

File: access_card/src/main/res/layout/layout_member_access_card.xml [hunk 1]
```diff
@@ -34,14 +35,28 @@
...
+        android:lines="1"
+        app:layout_constraintEnd_toStartOf="@id/reciprocalMember"
         app:layout_constraintHorizontal_bias="0"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/membershipType"
-        tools:text="Member Plus" />
+        tools:text="Member Plus Member Plus Member Plus Member Plus" />
+
+    <ImageView
+        android:id="@+id/reciprocalMember"
```
Comment (Cliabhach) [inline access_card/src/main/res/layout/layout_member_access_card.xml]: Shouldn't the default visibility be `GONE`? How are we hiding it if the membership is not reciprocal?

File: access_card/src/main/java/edu/artic/accesscard/AccessMemberCardFragment.kt [hunk 2]
```diff
@@ -167,6 +167,10 @@ class AccessMemberCardFragment : BaseViewModelFragment<AccessMemberCardViewModel
                 .map { it.size > 1 }
                 .bindToMain(switchCardHolder.visibility())
                 .disposedBy(disposeBag)
+
+        viewModel.isReciprocalMemberLevel
+                .bindToMain(reciprocalMember.visibility())
```
Comment (Cliabhach) [inline access_card/src/main/java/edu/artic/accesscard/AccessMemberCardFragment.kt]: What do you think of assigning the `memberInfo.memberLevel` as contentDescription?

File: access_card/src/main/res/layout/layout_member_access_card.xml (hunk 1 above)
Comment (sam33rdhakal) [reply access_card/src/main/res/layout/layout_member_access_card.xml]: Yes, it should be `GONE`. Updated.

File: access_card/src/main/java/edu/artic/accesscard/AccessMemberCardFragment.kt (hunk 2 above)
Comment (sam33rdhakal) [reply access_card/src/main/java/edu/artic/accesscard/AccessMemberCardFragment.kt]: Done.

**Pull request merged**